src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -24,6 +24,27 @@
 AuthEntity = NewType('AuthEntity', str)
 
 
+def get_auth_entity(daemon_type: str, daemon_id: str, host: str = "") -> AuthEntity:
+    """
+    Map the daemon id to a cephx keyring entity name
+    """
+    # despite this mapping entity names to daemons, self.TYPE within
+    # the CephService class refers to service types, not daemon types
+    if daemon_type in ['rgw', 'rbd-mirror', 'cephfs-mirror', 'nfs', "iscsi", 'ingress']:
```

Comment:
```suggestion
    if daemon_type in ['rgw', 'rbd-mirror', 'cephfs-mirror', 'nfs', 'iscsi', 'ingress']:
```
NIT, no reason to use double quotes for that one specific type